manager-bundle/src/ContaoManager/Plugin.php

```diff
@@ -77,12 +77,12 @@ public function getBundles(ParserInterface $parser): array
     {
         $configs = [
             BundleConfig::create(FrameworkBundle::class),
-            BundleConfig::create(SecurityBundle::class)->setLoadAfter([FrameworkBundle::class]),
+            BundleConfig::create(SecurityBundle::class)->setLoadAfter([FrameworkBundle::class, ContaoCoreBundle::class]),
```

Comment:
I guess this fixes the issue that I encountered: "the service parameter '%contao.backend.route_prefix%' is not defined" 🙈

Reply:
exactly 🙃